src/bin/server/accounts/mod.rs

```diff
@@ -173,7 +173,7 @@ impl AccountsHandler {
         let initial_account = Account::new();
 
         if sender.eq(&receiver) {
-            warn!(?sender, "transfer to themself");
+            warn!(?sender, "transfer to themselves");
```

Comment:
```suggestion
            warn!(?sender, "transfer to itself");
```

Reply:
Haaa, it's the third time I'm reading the grammar article you sent me, I still don't apply it correclty :sweat: